lib/apm.js

```diff
@@ -0,0 +1,21 @@
+// Copyright IBM Corp. 2014,2016. All Rights Reserved.
+// Node module: strong-supervisor
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+'use strict';
+
+var path = require('path');
+
+// Default paths for APM installation
+var apmLinuxPath = path.join('/opt', 'ibm', 'apm', 'agent');
+var apmWindowsPath = path.join('C:', 'IBM', 'APM');
```

Comment:
does this do the right thing? result in `'c:\\\\IBM\\APM'`? a direct string is likely fine here, to